.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "web5-spec"]
+	path = web5-spec
+	url = git@github.com:TBD54566975/web5-spec.git
```

Comment:
I think probably we should include some helpful info in the README for submodules, maybe inspiration from https://github.com/TBD54566975/web5-js/?tab=readme-ov-file#cloning

Another idea is to add a Justfile recipe